Northstar.CustomServers/mod/scripts/vscripts/mp/_base_gametype.gnut

```diff
@@ -1412,6 +1412,32 @@ void function CodeCallback_WeaponFireInCloak( entity player )
 		//player.SetCloakFlicker( 1.0, 2.0 )
 
 		DisableCloak( player, 0.5 )
+		entity cloakWeapon
+		int offhandSlot
+		for ( int i = 0; i <= OFFHAND_MELEE; i++ ) // OFFHAND_MELEE is the largest
```

Comment:
could you iterate over player.GetOffhandWeapons() instead here?

Reply:
iterate over player.GetOffhandWeapons() can't get cloak's exact slot, as for OFFHAND_ANTIRODEO it will return a `1` instead of `2`

Reply:
iterate over player.GetOffhandWeapons() can't get cloak's exact slot, if modders have a empty offhand slot for players